docs/application/vstools/tools/manifest-editor.md

```diff
@@ -369,11 +424,12 @@ This element contains specific service provider or user account protocol-related
 
 #### <icon> Element
```

Comment:
In Web browsing, it looks like `Element`. The page considered `<icon>` as a tag and removed it.

Reply:
Done